package.json

```diff
@@ -72,6 +72,7 @@
       "@tanstack/router-plugin": "workspace:*",
       "@tanstack/router-vite-plugin": "workspace:*",
       "@tanstack/react-router-with-query": "workspace:*",
+      "@tanstack/react-router-with-zod": "workspace:*",
```

Comment:
quick discussion of naming of the library: 

form uses this pattern: @tanstack/zod-form-adapter

should we use a similar one?

Reply:
I'm not a fan of the adapter name being before the project name tbh. 

I'd be more inclined towards `router-zod-adapter`.

Reply:
My thought process is we already named -with-query like this

Reply:
I prefer the adapter one